kumascript/macros/CSSRef.ejs

```diff
@@ -746,7 +746,159 @@ const text = mdn.localStringMap({
         'Legacy_layout_methods': 'Legacy layout methods',
         'Supporting_older_browsers': 'Supporting older browsers',
         'Assessment_Fundamental_Layout_Comprehension': 'Assessment: Fundamental Layout Comprehension',
+  },
+  'ko': {
```

Comment:
Rather than adding the `ko` key at the end, let's add it at the right position alphabetically, between `ja` and `ru`. 

This also holds for the other sidebars. 🙏

Reply:
I'll work on this now!